### DISCUSSION TITLE
Pow-K+ module integrated with Apple Home

### DISCUSSION THREAD

**Discussion by Andy46cap** [UtilitechAS/amsreader-firmware#650] [Q&A]
Hello ..
Have anyone some experience with Pow-K+ and Apple Home ?

**Comment by ArnieO** [comment]
Thank you for starting a potentially interesting discussion.
However, what I find does not seem to be feasible for our products, at least not in our current modest-scale production volume. 

It is technically possible to add code to our project to use the the Apple Homekit environment / connect to Apple Home.
However, linking a commercial product to the Apple Homekit environment requires an Apple license, see below.

So my current conclusion is that we will not go there at this time. 
**********************************

This is what I find:

I find this project on Espressif's Github: https://github.com/espressif/esp-apple-homekit-adk
It is _"a port of Apple's Open Source [HomeKit ADK](https://github.com/apple/HomeKitADK) for ESP32 and ESP32S2. This can be used by any developer to prototype **non-commercial** smart home accessories."_

This Espressif page https://www.espressif.com/en/news/ESP_Apple_HomeKit_ADK
says 
_"Espressif’s Apple HomeKit Application Development Kit (ADK) can be[ used under the same terms](https://www.espressif.com/en/products/sdks/esp-homekit-sdk) as Apple’s original open-source ADK, which was meant as a tool for hobbyists and other developers experimenting with the platform. However, there is a route to commercial licensing. **Anyone who is interested in commercializing their creations should register with Apple’s MFi licensing program** first and, then, [contact Espressif for more details on how to complete the process.](https://www.espressif.com/en/contact-us/sales-questions)"_

Information on Apple MFI: 
https://mfi.apple.com/
https://mfi.apple.com/en/how-it-works

![image](https://github.com/UtilitechAS/amsreader-firmware/assets/10295178/3d0332ef-87cd-464d-8e19-092b838d8cc2)
